server/server.vala

```diff
@@ -390,6 +390,16 @@ namespace Frida.Server {
 				connection.unregister_object (registration_id);
 			}
 
+			private void schedule_idle (owned ScheduledFunc function) {
```

Comment:
Nit: perhaps we should name the variable `func` to match the delegate name. In general I try to make variable names no more verbose than their corresponding types, so either we should rename `ScheduledFunc` to `ScheduledFunction`, or make `function` just `func`. As GLib uses `FooFunc` in a few places I tend to stick to that.